src/ConstructStyleSheet.js

```diff
@@ -23,37 +23,16 @@ const cssStyleSheetNewMethods = ['replace', 'replaceSync'];
 export function updatePrototype(proto) {
   cssStyleSheetNewMethods.forEach(methodKey => {
     proto[methodKey] = function () {
-      return ConstructStyleSheet.prototype[methodKey].apply(this, arguments);
+      /* This matches Chrome's behavior. Try running this:
+           var style = document.createElement('style');
+           document.head.appendChild(style);
+           style.sheet.replace('body { color: blue }');
+      */
+     return Promise.reject(
```

Comment:
This should only return a `Promise` for `CSSStyleSheet.prototype.replace`, `replaceSync` should only throw.

Reply:
Doh, good catch.